### DISCUSSION TITLE
Master 코스 9월 진행계획

### DISCUSSION THREAD

**Discussion by Jsueeee** [euphony-io/check-in-listener#3] [Show and tell]
# 앱개발 (개인안심번호)
- 코틀린, MVVM **`장수연`**
    - ViewModel 구현하기
- 리스너 구현하기 (수신 버튼) **`이진산`** , **`홍영석`**
- (데이터 저장) Room **`옥은진`**  **`장수연`**
    - dao 파일 만들기
    - db 파일 만들기
    - 데이터 클래스 파일
    - 데이터 저장하기
- csv 파일 추출 (전송버튼) **`김수진`**
    - 룸에서 데이터 가져오기
    - 텍스트 파일 추출하기

# 성능향상계획
- 코드 분석하기 **`~9/23`**
- 속도 향상 도전하기 **`~9/30`**
- rx 의 신뢰성 향상
- 데이터타입 변환 함수 추가

@euphony-io/eu-us

**Comment by designe** [comment]
[KIPASS-안내서-보건복지부.pdf](https://github.com/euphony-io/check-in-listener/files/7178472/KIPASS-.-.pdf)
KIPASS에 꼭 도입이 되었으면 좋겠군요 :)
